models/cube.js

```diff
@@ -50,20 +63,8 @@ const cubeSchema = mongoose.Schema({
     type: [String],
     default: [],
   },
-  cards: [
-    {
-      tags: [String],
-      finish: { type: String, default: 'Non-foil' },
-      status: String,
-      colors: [String],
-      cmc: Number,
-      cardID: String,
-      type_line: String,
-      addedTmsp: Date,
-      imgUrl: String,
-      details: {},
-    },
-  ],
+  cards: [Card],
+  maybe: [Card],
```

Comment:
I'd like to include default values for these

Reply:
done